packages/assets-controllers/CHANGELOG.md

```diff
@@ -13,6 +13,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Add token detection support
   - Add NFT detection support
 
+### Changed
+
+- **UPDATE:** Refactor `TokenRatesController` to support processing multiple chains simultaneously. The controller now accepts an array of chain IDs and tickers instead of a single value, streamlining the polling process by iterating over all chains in one loop ([#5645](https://github.com/MetaMask/core/pull/5645))
```

Comment:
Nit: What do you think about ?
```suggestion
- Refactor `TokenRatesController` to support processing multiple chains simultaneously ([#5645](https://github.com/MetaMask/core/pull/5645))
  - The controller now accepts an array of chain IDs and tickers instead of a single value, streamlining the polling process by iterating over all chains in one loop